locales/en-US/experiments.ftl

```diff
@@ -143,7 +142,6 @@ snoozetabsContributors5Title = Firefox QA
 snoozetabsContributors6Title = Softvision QA
 snoozetabsContributors7Title = Softvision QA
 tabcenterDescription = What would it be like to move tabs from the top of the browser to the side? We wanted to find out!
-tabcenterWarning = Firefox is changing fast. Tab Center may have bugs in Firefox 55 and will be disabled completely in Firefox 56+.
```

Comment:
We just added this one…
https://github.com/lmorchard/testpilot/commit/842fe486715b9c0c6b56d3451856d6f3aa64dbac

Issue with the extraction?